### PR TITLE
Added hand drawn tile: (The maze inside the maze)

### PR DESCRIPTION
**Pull request by saarthakchats**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a description of change below.
- [x] Squashed the commits.

### Screenshots,
screenshot 51

### CODE REVIEW

**Comment by saarthakchats** [comment]
@niccokunzmann

**Comment by niccokunzmann** [comment]
Thanks!

**Comment by saarthakchats** [comment]
Thanks

File: js/levels.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ function createForestLevel() {
 function createDesertLevel() {
   return new Level("Desert", [
     [desert.none, desert.right, desert.right, desert.right, desert.right, desert.none],
-    [desert.none, desert.top, desert.both, desert.both, desert.both, desert.both],
+    [desert.drawnbyme, desert.top, desert.both, desert.both, desert.both, desert.both],
```
Comment (niccokunzmann) [inline js/levels.js]: this will break I guess, since you added it to door.drawnbyme, it is not available for desert.drawnbyme.

**Pull request merged**